setup.py

```diff
@@ -20,12 +20,14 @@
 
 PACKAGE_ROOT = os.path.abspath(os.path.dirname(__file__))
 
+version = "0.24.0"
+
 with io.open(os.path.join(PACKAGE_ROOT, "README.rst")) as file_obj:
     README = file_obj.read()
 
 setup(
     name="gapic-generator",
-    version="0.24.0",
+    version=version,
```

Comment:
If you feel strongly about the version string being moved, we can also edit the bot so it respects the spacing around the '='. https://github.com/googleapis/release-please

Reply:
No strong feelings in the slightest.